Tools/setup/Dockerfile

```diff
@@ -0,0 +1,55 @@
+#
+# PX4 base development environment
+#
+
+FROM ubuntu:20.04
+LABEL maintainer="Daniel Agar <daniel@agar.ca>, Ramon Roche <mrpollo@gmail.com>"
+
+ENV DEBIAN_FRONTEND noninteractive
+ENV LANG C.UTF-8
+ENV LC_ALL C.UTF-8
+
+# Installing required utilities
+RUN apt-get update && apt-get -y --quiet --no-install-recommends install \
+  ca-certificates \
+  gnupg \
+  lsb-core \
+  lsb-release \
+  sudo \
+  software-properties-common \
+  wget \
+  gosu \
+  ;
+
+# Install PX4 Requirements
+COPY Tools/setup/requirements.txt /tmp/requirements.txt
+COPY Tools/setup/ubuntu.sh /tmp/ubuntu.sh
+# We support pre-downloading the gcc arm none eabi compiler
+# to speed up build times, if the file is not present when
+# building, the ubuntu.sh script will download it from source
+COPY *gcc-arm-none-eabi-9-2020-q2-update-linux.tar.bz2 /tmp/gcc-arm-none-eabi-9-2020-q2-update-linux.tar.bz2
```

Comment:
This line fails if the file doesn't exist:

```
Step 9/22 : COPY *gcc-arm-none-eabi-9-2020-q2-update-linux.tar.bz2 /tmp/gcc-arm-none-eabi-9-2020-q2-update-linux.tar.bz2
COPY failed: no source files were specified
```

Reply:
I verified this morning on a new build w/o the file present, and my build passes. I wonder why you get a fail. Can you please share more about your setup?

Reply:
I'm using a fresh install of Docker on a Pop!OS 22.04 machine.

```
$ uname -a
Linux pop-os 5.17.15-76051715-generic #202206141358~1655919116~22.04~1db9e34 SMP PREEMPT Wed Jun 22 19 x86_64 x86_64 x86_64 GNU/Linux
$ docker --version
Docker version 20.10.17, build 100c701
``

Reply:
I'm on the same version, the only difference is I'm running on macOS, regardless I just changed the line

```
$ docker --version
Docker version 20.10.12, build e91ed57
```